asmcomp/cmm_helpers.ml

```diff
@@ -805,14 +805,14 @@ let make_alloc_generic set_fn dbg tag wordsize args =
     | e1::el -> Csequence(set_fn (Cvar id) (Cconst_int (idx, dbg)) e1 dbg,
                           fill_fields (idx + 2) el) in
     Clet(VP.create id,
-         Cop(Cextcall("caml_alloc", typ_val, true, None),
+         Cop(Cextcall("caml_alloc", typ_val, [XInt;XInt], true, None),
```

Comment:
I wonder if we might write `[]` instead of `[XInt;Xint]` here, as in other places.

Reply:
Well spotted.  Done.